### PR TITLE
Speeds up condition score retrieval via Postgis

### PR DESCRIPTION
**Pull request by strangeties**
Condition score retrieval now takes ~6-8 secs.

TODO: see whether this can be improved further by sending a list of condition raster names rather than a single one.

Results seem consistent with the prior python solution in #320.
image

### CODE REVIEW

**Review by thequeenofspades — APPROVED** [review]

**Review by riecke — APPROVED** [review]
This LG, but please raise a bug to add a unit test for this.

File: src/planscape/plan/views.py [hunk 1]
```diff
@@ -297,42 +301,39 @@ def get_project_areas(request: HttpRequest) -> HttpResponse:
...
-                conditions.append(
-                    {'condition': str(c), 'mean_score': str(accumulator.stats[c]
-                     ['sum'] / accumulator.stats[c]['count'])})
-
-        response = {'conditions': json.dumps(conditions)}
-        return HttpResponse(
-            JsonResponse(response),
-            content_type='application/json')
+        with connection.cursor() as cursor:
+            plan = get_plan_by_id(request.GET)[0]
```
Comment (elsiel23) [inline src/planscape/plan/views.py:305]: can you add user validation here to check that the user is allowed to access the plan? i have a PR that does this that ill send out for review soon with examples!

Comment (strangeties) [reply src/planscape/plan/views.py:305]: I see #354.
I'll submit this PR as-is and follow up with user validation once #354 has been submitted.

**Comment by strangeties** [comment]
thanks for reviewing

**Comment by thequeenofspades** [comment]
Friendly reminder to add this endpoint to the wiki :) https://github.com/OurPlanscape/Planscape/wiki/Backend-API

**Pull request merged**